src/en/docs/install.md

```diff
@@ -9,5 +9,7 @@ There are several ways to create an instance, so please choose one below and rea
 [See here](./install/bash.html)
 
 ## Manually
-
 [See here](./install/manual.html)
+
+## Using Yunohost
+[See here](https://github.com/YunoHost-Apps/misskey_ynh) 
```

Comment:
I think in that case it would be nicer to write something like this:

```suggestion
Misskey can be installed as an App on YunoHost. For more information see [the YunoHost repository](https://github.com/YunoHost-Apps/misskey_ynh).
```